projects/packages/forms/src/dashboard/integrations/akismet-card.tsx

```diff
@@ -65,15 +65,20 @@ const AkismetDashboardCard = ( {
 							}
 						) }
 					</p>
-					<Button
-						variant="secondary"
-						href={ settingsUrl }
-						target="_blank"
-						rel="noopener noreferrer"
-						__next40pxDefaultSize={ true }
-					>
-						{ __( 'Add Akismet key', 'jetpack-forms' ) }
-					</Button>
+					<HStack spacing="3" justify="start">
+						<Button
+							variant="secondary"
+							href={ settingsUrl }
+							target="_blank"
+							rel="noopener noreferrer"
```

Comment:
`Button` might add these on its own when it has `target="_blank"` but I haven't checked.

Reply:
Just double checked this. It's possible I'm missing something, but I don't see any indication in the code for the component that these would be added. So for now, I will leave them in place.